template_credentials.go

```diff
@@ -0,0 +1,78 @@
+package transloadit
+
+import (
+	"context"
+)
+
+// Template contains details about a single template.
+type TemplateCredential struct {
+	ID          string                 `json:"id"`
+	Name        string                 `json:"name"`
+	Type        string                 `json:"type"`
+	Content     map[string]interface{} `json:"content"`
+	Created     string                 `json:"created",omitempty`
+	Modified    string                 `json:"modified",omitempty`
+	Deleted     string                 `json:"deleted",omitempty`
+	Stringified string                 `json:"stringified",omitempty`
```

Comment:
Should we actually expose Deleted and Stringified? Are they of any use? /cc @kvz

Reply:
@Acconut , @kvz, I put the 2 fields as, for me, a sdk should be the image of the API as it can be used by everybody. 
If you prefer,I can remove the 2 fields and if a user asked for the fields we will add them.

Reply:
I think we should actually remove Deleted from the API, good call. Not sure about Stringifed, @tim-kos ?